scripts/install-golang.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+#  with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+architecture=""
+case $(uname -m) in
+    x86_64)
+        architecture="amd64"
+        ;;
+    arm64)
+        architecture="arm64"
+        ;;
+    aarch64)
+        architecture="arm64"
+        ;;
+    *)
+        echo $"Unknown architecture $0"
+        exit 1
+esac
+
+if [ "$architecture" == "amd64" ]; then export GOARCH=amd64; fi
+if [ "$architecture" == "arm64" ]; then export GOARCH=arm64; fi
+
+export GOROOT=/usr/local/go
+export PATH=$PATH:$GOROOT/bin
+
+# if golang isn't installed then we'll install it
+if [ -d "$GOROOT" ]; then
```

Comment:
$GOROOT might be pre-initialized in codebuild buildspecs (for rpm builds). Maybe we can check if go exists first and then export this